install_config/install/advanced_install.adoc

```diff
@@ -617,6 +620,26 @@ value; you only need to set this if you want your `git clone` operations to use
 a different value.
 |===
 
+[[advanced-install-configuring-firewalls]]
+=== Configuring the Firewall
+
+{product-title} uses iptables as the default firewall, but you can switch to use firewalld in the install.
+
+Because iptables is the default filewall, {product-title} is designed to have it
+configured automatically. However, iptables rules can break {product-title} if
+not configured correctly. The advantages of firewalld include allowing multiple
+objects to safely share the firewall rules. However, it is not available on RHEL
+Atomic Host.
+
+To use firewalld as the firewall for an {product-title} install, add the
+`os_firewall_use_firewalld` variable to the list of configuration variables in
+Ansible host file on install:
+
+----
+[OSEv3:vars]
+os_firewall_use_firewalld=True
+----
+
```

Comment:
@sdodson: I thought there were two problems:

1) Atomic Host didn't have firewalld (but that will be fixed soon)
2) If you added a node to an existing cluster, the old nodes would have raw iptables, but the new would default to firewalld meaning you would end up with a mixed cluster

I thought 2 was the reason we reverted.  But if you are installing a new cluster it would not be a problem.

@danwinship @dcbw @openshift/networking -- What are your thoughts on this?